Composer/packages/client/package.json

```diff
@@ -20,6 +20,7 @@
     "@bfc/extensions": "*",
     "@bfc/indexers": "*",
     "@bfc/shared": "*",
+    "@bfcomposer/bf-lu": "^1.1.6",
```

Comment:
Is this version contains the patch that make section id stable?

Reply:
yes